### PR TITLE
:test_tube: Add new textfield test

### PR DESCRIPTION
**Pull request by MarioBatalha**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1207089662029583

### CODE REVIEW

**Review by KipandaJr — CHANGES_REQUESTED** [review]
- Check the point on aria-label

**Review by KipandaJr — CHANGES_REQUESTED** [review]
- Check if the value that u put on input is expected
- Check the description test

File: src/lib/components/text-field/stories/index.stories.tsx [hunk 1]
```diff
@@ -25,29 +25,59 @@ export const Default: Story = {
...
 
-    await step('Check property value and args', () => {
-      expect(border.trim()).toBe('0px none rgb(0, 0, 0)');
-      expect(background.substring(0, 16)).toBe('rgba(0, 0, 0, 0)');
-      expect(color.trim()).toBe('rgb(0, 0, 0)');
-      expect(textAlignment).toBe('start');
-      expect(args.placeholder).toBe('0.123');
+    await step('Testing textfield input', async () => {
+      const inputField = textfield.children[0].children[0].children[0];
+      const inputValue = '0.123';
```
Comment (KipandaJr) [inline src/lib/components/text-field/stories/index.stories.tsx:46]: fake

**Review by KipandaJr — CHANGES_REQUESTED** [review]

File: src/lib/components/text-field/stories/index.stories.tsx [hunk 2]
```diff
@@ -25,29 +25,66 @@ export const Default: Story = {
...
+      ).toHaveStyle('background-color: rgba(0, 0, 0, 0)');
     });
 
-    await step('Check property value and args', () => {
-      expect(border.trim()).toBe('0px none rgb(0, 0, 0)');
-      expect(background.substring(0, 16)).toBe('rgba(0, 0, 0, 0)');
-      expect(color.trim()).toBe('rgb(0, 0, 0)');
-      expect(textAlignment).toBe('start');
-      expect(args.placeholder).toBe('0.123');
+    await step('Testing textfield input', async () => {
```
Comment (atilasousa) [inline src/lib/components/text-field/stories/index.stories.tsx:46]: it should be Testing Textfield structure

File: src/lib/components/text-field/index.tsx [hunk 3]
```diff
@@ -121,6 +120,7 @@ export const TextField: FC<PropsWithRef<TextFieldProps>> = forwardRef(
             flexDirection="column"
             justifyContent="center"
             mr={status ? '0.5rem' : 'unset'}
+            aria-label="input"
```
Comment (atilasousa) [inline src/lib/components/text-field/index.tsx:123]: Check this aria-label

**Review by atilasousa — CHANGES_REQUESTED** [review]

**Review by atilasousa — APPROVED** [review]

**Review by atilasousa — APPROVED** [review]

**Review by KipandaJr — APPROVED** [review]

**Pull request merged**